examples/data-sources/jira_field/by_id/data-source.tf

```diff
@@ -0,0 +1,11 @@
+data "jira_field" "example" {
+  id = "customfield_10001"
+}
+
+output "field_name" {
+  value = data.jira_field.example.field_name
+}
+
+output "field_type" {
+  value = data.jira_field.example.field_type
+}
```

Comment:
Missing newline at end of file. Please add a newline character at the end of this file.